### PR TITLE
Migrated repository selection from `builder.sh` to `wazuh-install.sh`

### PR DESCRIPTION
**Pull request by davidcr01**
## Description

Closes: #63 
The aim of this PR is to migrate the repository selection performed in the `builder.sh` script to the `wazuh-install.sh` script.

This change impacts on the QA modules: https://github.com/wazuh/wazuh-qa/issues/5760 and this PR should be merged synced with the QA team.

## Testing

- The installation testing and validations were done here: https://github.com/wazuh/wazuh-installation-assistant/issues/63#issuecomment-2361293872
- The GHAs were tested here: https://github.com/wazuh/wazuh-installation-assistant/issues/63#issuecomment-2363948220

### CODE REVIEW

File: install_functions/installVariables.sh [hunk 1]
```diff
@@ -7,11 +7,18 @@
 # Foundation.
 
 ## Package vars
-readonly wazuh_major="4.10"
-readonly wazuh_version="4.10.0"
+readonly wazuh_major="4.9"
+readonly wazuh_version="4.9.0"
```
Comment (Enaraque) [inline install_functions/installVariables.sh:11]: The version of both variables must be 4.10 if this PR is for the `4.10.0` branch

File: install_functions/installMain.sh [hunk 2]
```diff
@@ -236,6 +262,9 @@ function main() {
 
     common_checkInstalled
     checks_arguments
+    if [ -n "${development}" ]; then
+        checks_filebeatURL
+    fi
```
Comment (Enaraque) [inline install_functions/installMain.sh:267]: Checking the Filebeat URL is done for all cases, not just for development, I think.

**Review by Enaraque — CHANGES_REQUESTED** [review]

File: install_functions/installVariables.sh (hunk 1 above)
Comment (davidcr01) [reply install_functions/installVariables.sh]: Done

File: install_functions/installMain.sh (hunk 2 above)
Comment (davidcr01) [reply install_functions/installMain.sh:267]: It is not necessary to check it in production case. Notice that:
`source_branch="v${wazuh_version}"`, then
`filebeat_wazuh_template="https://raw.githubusercontent.com/wazuh/wazuh/${source_branch}/extensions/elasticsearch/7.x/wazuh-template.json"
`
The template will be downloaded from the tag, then, it is not necessary to change the filebeat template URL

**Review by Enaraque — APPROVED** [review]

**Review by c-bordon — APPROVED** [review]

**Pull request merged**